Misc/NEWS.d/next/Documentation/2023-01-23-17-55-15.gh-issue-101273.Ae-Hed.rst

```diff
@@ -0,0 +1 @@
+Expand :mod:`enum` documentation on limitations when shadowing a built-in.
```

Comment:
We don't need news for docs :)

Reply:
Fixed in afde363e70c626697e04e141359413caff0d708e